src/FlywheelEventStoreAdapter.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Prooph\EventStore\Adapter\Flywheel;
+
+use ArrayIterator;
+use Iterator;
+use JamesMoss\Flywheel\Config;
+use JamesMoss\Flywheel\Document;
+use JamesMoss\Flywheel\Repository;
+use Prooph\Common\Messaging\Message;
+use Prooph\Common\Messaging\MessageConverter;
+use Prooph\Common\Messaging\MessageFactory;
+use Prooph\Common\Messaging\MessageDataAssertion;
+use Prooph\EventStore\Adapter\Adapter;
+use Prooph\EventStore\Exception\RuntimeException;
+use Prooph\EventStore\Stream\Stream;
+use Prooph\EventStore\Stream\StreamName;
+
+final class FlywheelEventStoreAdapter implements Adapter
+{
+    private $rootDir;
+    private $messageFactory;
+    private $messageConverter;
+
+    public function __construct($rootDir, MessageFactory $messageFactory, MessageConverter $messageConverter)
+    {
+        $this->rootDir = $rootDir;
```

Comment:
Is it really necessary since Flywheel will [create the directory](https://github.com/jamesmoss/flywheel/blob/master/src/JamesMoss/Flywheel/Repository.php#L39) if it doesn't exist yet?